src/app/redux/TransactionSaga.test.js

```diff
@@ -0,0 +1,253 @@
+/* global describe, it, before, beforeEach, after, afterEach */
+
+import chai, { expect } from 'chai';
+import { call, select } from 'redux-saga/effects';
+import { api } from '@steemit/steem-js';
+import {
+    preBroadcast_comment,
+    createPermlink,
+    createPatch,
+    watchForBroadcast,
+    watchForUpdateAuthorities,
+    watchForUpdateMeta,
+    watchForRecoverAccount,
+    preBroadcast_transfer,
+} from './TransactionSaga';
+import * as transactionActions from 'app/redux/TransactionReducer';
+import { DEBT_TICKER } from 'app/client_config';
+
+let operation = {
+    type: 'comment',
+    author: 'Alice',
+    body:
+        "The Body is a pretty long chunck of text that represents the user's voice, it seems they have much to say, and this is one place where they can do that.",
+    category: 'hi',
+    json_metadata: {
+        tags: ['hi'],
+        app: 'steemit/0.1',
+        format: 'markdown',
+    },
+    parent_author: 'candide',
+    parent_permlink: 'cool',
+    title: 'test',
+    __config: {},
+    errorCallback: () => '',
+    successCallback: () => '',
+    memo: '#testing',
+};
+
+const {
+    author,
+    category,
+    errorCallback,
+    successCallback,
+    parent_author,
+    parent_permlink,
+    type,
+    __config,
+    json_metadata,
+    title,
+    body,
+    memo,
+} = operation;
```

Comment:
for clarity i think it'd be better to refer back to `operation` for these items in the tests below